pkg/providers/tinkerbell/pbnj/client.go

```diff
@@ -0,0 +1,66 @@
+package pbnj
+
+import (
+	"context"
+	"os"
+
+	"github.com/tinkerbell/cluster-api-provider-tinkerbell/pbnj/client"
+	"github.com/tinkerbell/pbnj/api/v1"
+)
+
+const (
+	PbnjGrpcAuth = "PBNJ_GRPC_AUTHORITY"
+)
+
+type Pbnj struct {
+	pbnj *client.PbnjClient
+}
+
+type BmcSecretConfig struct {
+	Host     string
+	Username string
+	Password string
+	Vendor   string
+}
+
+func NewPBNJClient(pbnjGrpcAuth string) (*Pbnj, error) {
```

Comment:
Its very difficult to know what valid values for this are. Can we define some constants to help consumers?

Reply:
Customers are providing this value in the clusterconfig file (tinkerbellPBnJGRPCAuth) and I am passing it's value in this function only.

Reply:
Understood. If we forget about how this package is being used externally for a moment, how would we know whats valid and whats not valid? Does PbnJ define some consts upstream? If so, could we have a code comment that links to it.